src/components/PlantGallery/PlantGallery.tsx

```diff
@@ -1,11 +1,42 @@
+import usePhoto from 'utils/usePhoto';
+import { getPhotoLinks } from 'utils/flickr';
+import ImageContainer from 'components/Image/ImageContainer';
 import type { PlantType } from 'components/PlantFiche/PlantFiche';
 
 type PlantGalleryType = Readonly<{ plant: PlantType }>;
 
 export default function PlantGallery({ plant }: PlantGalleryType) {
+  const photo = usePhoto(plant.imageID[0]);
+  const links = photo ? getPhotoLinks(photo) : null;
+
   return (
-    <div className="flex flex-col items-center justify-center w-full h-full bg-blue-light">
-      plant gallery placeholder
+    <div className="flex flex-col items-center w-full h-full">
+      <div className="flex-1">
+        <div className="flex h-full items-center justify-center">
```

Comment:
These 3 flex dives feel like overkill for me, maybe there's a simpler way to place them accurately